src/test/java/com/wikia/webdriver/testcases/discussions/SharingTests.java

```diff
@@ -42,8 +43,7 @@ public void anonUserOnMobileCanSeeSocialNetworkIcons() {
   public void anonUserOnDesktopCanSeeSocialNetworkIconsInPost() {
     toggleShareIconClickDisplaysSocialNetworkIcons(expected_networks_for_english_language);}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)